packages/app/android/app/src/main/AndroidManifest.xml

```diff
@@ -3,8 +3,7 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.INTERNET" />
     <application
-        android:label="Meli"
-        android:name="${applicationName}"
+        android:label="${applicationLabel}"
```

Comment:
this is the displayed name of the app when installed. there's a couple of ways of going about this (involving using `resources/strings.xml` files) but figured that using a manifest placeholder that we define in our build config is more straightforward and sufficient for our needs